src/api256.rs

```diff
@@ -315,6 +315,60 @@ impl Api256 {
     }
 }
 
+#[wasm_bindgen]
+pub struct EncryptedSearch {
+    rng: Mutex<DefaultRng>,
+}
+
+#[wasm_bindgen]
+impl EncryptedSearch {
+    #[wasm_bindgen(constructor)]
+    pub fn new() -> EncryptedSearch {
+        // 1 MB
+        const BYTES_BEFORE_RESEEDING: u64 = 1 * 1024 * 1024;
```

Comment:
I think I'd go up to more than this. We trust chacha20 and it doesn't cycle for a very very long time. 10MB seems perfectly reasonable both here and in recrypt.

Reply:
Done